gui/src/2D/controls/control.ts

```diff
@@ -1617,24 +1617,32 @@ export class Control {
     }
 
     protected _evaluateClippingState(parentMeasure: Measure) {
+        let evaluatedMeasure = new Measure(0, 0, 0, 0);
+        this._currentMeasure.transformToRef(this._transformMatrix, evaluatedMeasure);
+
+        let evaluatedParentMeasure = new Measure(0, 0, 0, 0);
+        const parentMatrix = this.parent?._transformMatrix;
```

Comment:
You should move the 3 lines in the if block below as it already checks for parent nullity. preventing the need for the ? and parentMatrix test.